core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -123,7 +135,7 @@ export class RteHelperModalComponent {
     private contextService: ContextService,
     private focusManagerService: FocusManagerService,
     private imageLocalStorageService: ImageLocalStorageService,
-    private imageUploadHelperService: ImageUploadHelperService
+    private imageUploadHelperService: ImageUploadHelperService,
```

Comment:
Why change this? Avoid spurious changes here and elsewhere -- please check the "Files Changed" tab to ensure that the only changes there are the ones that need to be made.